Demos/Run-VMware-Demo.SVGAWorld.x86.bat

```diff
@@ -1,4 +1,4 @@
 cd %~dp0
 cd ..\bin
-start Mosa.Tool.Launcher.exe -autostart -include Source/Mosa.Demo.SVGAWorld.x86/Include -vmware -vmdk -output-map -output-asm -output-debug Mosa.Demo.SVGAWorld.x86.dll
+Mosa.Tool.Launcher.exe -autostart -include Include -vmware -vmdk -output-map -output-asm -output-debug Mosa.Demo.SVGAWorld.x86.dll
```

Comment:
I've changed where the include directory is located. Now, it's copied to the output directory, and the code uses ``Path.GetFullPath(IncludeDirectory)``. This made it easier to implement an include directory option in the GUI launcher, since it uses a file dialog with the absolute path.